modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -188,6 +189,32 @@ def validate_form_526_location_codes!
           end
         end
 
+        def validate_form_526_veteran_homelessness!
+          if too_many_homelessness_attributes_provided? || unnecessary_homelessness_point_of_contact_provided?
```

Comment:
nit: i would use go with a guard clause here personally. less nesting the easier to read imo

Reply:
I would too, but it's make the line too long.  😞